src/cudamatrix/cu-matrix.cc

```diff
@@ -1031,13 +1033,15 @@ void CuMatrixBase<Real>::SymAddMat2(
 #if HAVE_CUDA == 1
   if (CuDevice::Instantiate().Enabled()) {
     Timer tim;
-    char trans = (transA == kTrans ? 'N' : 'T');
+    cublasOperation_t trans = (transA == kTrans ? CUBLAS_OP_N : CUBLAS_OP_T);
     MatrixIndexT A_other_dim = (transA == kNoTrans ? A.num_cols_ : A.num_rows_);
-    
-    cublas_syrk('U', trans, num_rows_, A_other_dim, alpha, A.Data(),
+    cublasHandle_t handle = CuDevice::Instantiate().GetHandle();
+    cublasStatus_t stat;
+    stat=cublas_syrk(handle, CUBLAS_FILL_MODE_UPPER, trans, num_rows_, A_other_dim, alpha, A.Data(),
```

Comment:
Remember there is an 80-character length limits, you may have to split lines if you make them longer.
If you use emacs then check out my .emacs file, there is some relevant stuff in there e.g. calling google c style.